src/mobot_client/tests/test_models.py

```diff
@@ -0,0 +1,191 @@
+# Copyright (c) 2021 MobileCoin. All rights reserved.
+import unittest
+from typing import List, Dict
+from collections import defaultdict
+from django.test import TestCase
+import factory.random
+from mobot_client.tests.factories import *
+
+from mobot_client.models import (Drop,
+                                 DropSession,
+                                 Sku,
+                                 DropType,
+                                 BonusCoin,
+                                 Order,
+                                 OrderStatus,
+                                 OutOfStockException)
+from mobot_client.models.states import SessionState
+
+factory.random.reseed_random('mobot cleanup')
+
+
+class ModelTests(TestCase):
+
+    def test_items_available(self):
```

Comment:
Could we get a one-sentence doctest for each test describing what it is testing?